_posts/2019-12-18-two-monitors.md

```diff
@@ -0,0 +1,33 @@
+# Two Monitors
+
```

Comment:
Decades... I have no idea where is the other part of this but you posted with no review and you got typos.

If you see I didn't rewiew something just S A Y

Reply:
If you don't review in a timely manner and I have no reviewed matertial to post obviously I need to post non reviewed material.

What decades?

And I don't see any typo nor anything in the review related to any typo here... ?

Reply:
@MsSlacker ^